client/src/graphs/plugins/history/types.ts

```diff
@@ -80,11 +80,36 @@ export type EditRecord = {
   affectedItems: GEdgeLabelEditRecord[];
 };
 
+/**
+ * a record indicating the graph state was loaded
```

Comment:
maybe something about how it facilitates both adding and removing nodes and edges in a single record. Loading is the verb we use in base api to describe an action that must require history plugin to have a record for adding and removing in a single record